ganga/GangaCore/Lib/Condor/Condor.py

```diff
@@ -797,4 +797,38 @@ def timedetails(self):
     updateMonitoringInformation = \
         staticmethod(updateMonitoringInformation)
 
+    def peek(self, filename=None, command=None):
+        job = self.getJobObject()
+
+        if not self.id:
+            logger.warning("Job %s not running" % job.id)
+            return False
```

Comment:
We should use the job status to decide on this. But is the check actually required? Don't we only end up here if the job is in the `running` state?
```suggestion
        if job.status!='running':
            logger.warning("Job %s not running" % job.id)
            return False
```

Reply:
I will remove this as it may not be required.